<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotcopyDlg</class>
 <widget class="QWidget" name="HotcopyDlg">
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout" name="m_MainGrid">
     <item row="1" column="0">
      <widget class="QLabel" name="m_Destlabel">
       <property name="text">
        <string>Destination folder:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlRequester" name="m_DestpathEditor">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_Srclabel">
       <property name="text">
        <string>Repository to copy:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KUrlRequester" name="m_SrcpathEditor">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_Cleanlogs">
     <property name="text">
      <string>Clean logs</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_SrcpathEditor</tabstop>
  <tabstop>m_DestpathEditor</tabstop>
  <tabstop>m_Cleanlogs</tabstop>
 </tabstops>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
